migrations/2019-09-11-164500/up.sql

```diff
@@ -1,11 +1,54 @@
-alter table `batches` change column `user_id` `userid` int(11) not null;
-alter table `batches` change column `collection_id` `collection` int(11) not null;
-alter table `bso` change column `user_id` `userid` int(11) not null;
-alter table `bso` change column `collection_id` `collection` int(11) not null;
-alter table `bso` change column `expiry` `ttl` bigint(20) not null;
-alter table `user_collections` change column `user_id` `userid` int(11) not null;
-alter table `user_collections` change column `collection_id` `collection` int(11) not null;
-alter table `user_collections` change column `modified` `last_modified` bigint(20) not null;
--- must be last in case of error
--- the following column is not used, but preserved for legacy and stand alone systems.
-alter table `bso` add column `payload_size` int(11) default 0;
+-- tests to see if columns need adjustment
```

Comment:
For historical purposes, would we want to create a new file each time we make changes here vs changing the single file that's timestamped to `2019-09-11`?

Reply:
Normally, yes. However this file breaks existing migration and needs to be fixed. 

I'll note that we also do not provide `down.sql` in all cases, meaning that some changes are effectively irreversible. This is rarely needed, since code would also have to be regressed.